posthog/api/async_migration.py

```diff
@@ -78,6 +82,15 @@ def trigger(self, request, **kwargs):
         trigger_migration(migration_instance)
         return response.Response({"success": True}, status=200)
 
+    @action(methods=["POST"], detail=True)
+    def resume(self, request, **kwargs):
+        migration_instance = self.get_object()
+        resumable, error = is_migration_resumable(migration_instance)
+        if error:
+            return response.Response({"success": False, "error": error,}, status=400,)
```

Comment:
I love this kind of error handling <3
![image](https://user-images.githubusercontent.com/391319/148125756-5a5eabeb-d81e-4a35-a233-8c20d7bb89e7.png)

Should this be a `400` or should it be a `500`? Feels like a `500` to me.

Reply:
```suggestion
        resumable, error = is_migration_resumable(migration_instance)
        if error:
            return response.Response({"success": False, "error": error,}, status=500,)
```